cms/static/coffee/spec/main.coffee

```diff
@@ -221,8 +221,11 @@ define([
     "coffee/spec/models/settings_grading_spec", "coffee/spec/models/textbook_spec",
     "coffee/spec/models/upload_spec",
 
-    "coffee/spec/views/metadata_edit_spec", "coffee/spec/views/module_edit_spec",
-    "coffee/spec/views/textbook_spec", "coffee/spec/views/upload_spec",
+    "coffee/spec/views/course_info_spec",
```

Comment:
Good question. The other two files were moved beneath common and I verified that the specs were registered, so it was just that one test file that got lost.